docker-compose/README.md

```diff
@@ -5,14 +5,17 @@ This directory provides two ways to start the JSON API and Stargate coordinator
 ## Stargate JSON API with 3-node DSE 6.8 cluster
 
 > **PREREQUISITES:** Please build the latest JSONAPI docker image first by running: 
+> 
 > ```bash
 > ./mvnw clean package -Dquarkus.container-image.build=true -DskipTests
 > ```
+> 
+> You can control the platform using the `-Dquarkus.docker.buildx.platform=linux/amd64,linux/arm64` property.
 
 You can start a simple Stargate configuration with the following command:
 
 ```
-docker compose up -d
+docker-compose up -d
```

Comment:
Question on this: both work, I think, but I was not 100% sure which one is recommended approach. F.ex:

https://stackoverflow.com/questions/66514436/difference-between-docker-compose-and-docker-compose

seems to suggest `docker compose` is the "new" way and `docker-compose` being deprecated.

So maybe it should be left as-is? I'll ask on our Slack channel just so we know which one to go with.

Reply:
That's again that v1 vs v2 story.. Let's do whatever you want, I am fine with both.. I did remove that `--wait` before to make it work with v1, so I wanted to make this as well..

Reply:
Ok let's see if we can figure out one to use. Not a huge thing don't want to make it one. Will approve in the meantime.